src/coordination/include/coordination/coordinator_handlers.hpp

```diff
@@ -44,6 +48,49 @@ class CoordinatorHandlers {
 
   static void GetDatabaseHistoriesHandler(replication::ReplicationHandler &replication_handler, slk::Reader *req_reader,
                                           slk::Builder *res_builder);
+
+  template <typename T>
```

Comment:
Can you rename to TResponse or similar to it is easier to reason what template replaces